src/main/java/in/projecteka/consentmanager/dataflow/model/hip/DataRequest.java

```diff
@@ -0,0 +1,19 @@
+package in.projecteka.consentmanager.dataflow.model.hip;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import java.util.UUID;
+
+@Getter
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode
+public class DataRequest {
+    private final String transactionId;
```

Comment:
I think transaction Id should be UUID.. not sure.. please confirm it once agin from contract

Reply:
in contract schema, it is string

Reply:
changed it to UUID